templates/profile/prometheus/exporter/node/rubyHealthCheck-testall.sh.erb

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#Managed By Puppet
+#
+
+cd /usr/local/rubytests/
+export PATH=/l/local/rbenv/shims:/l/local/rbenv/bin:/usr/local/bin:/usr/local/sbin:$PATH
+export RBENV_ROOT=/l/local/rbenv
```

Comment:
We should pull these out to the cronjob itself so they apply for anything we run. We're mostly interested in `rbenv` and `bundle`, which are placed on the path by bin and shims, respectively. Then we can invoke them directly.

Reply:
added to cronjob, rather than testall script.